src/database/sql_database.cc

```diff
@@ -1143,13 +1147,13 @@ int SQLDatabase::createContainer(int parentID, std::string name, const std::stri
     std::string dbLocation = addLocationPrefix((isVirtual ? LOC_VIRT_PREFIX : LOC_DIR_PREFIX), virtualPath);
 
     auto fields = std::vector {
```

Comment:
I'm not sure whether `fmt::join` works with `SQLIdentifier`. But in case, you could make the fields vector a `vector<SQLIdentifier>` and updated the insert method.

Reply:
I'm just getting back to where I was before my vaccation - and I am pretty sure that I successfully tested this before. Today, I managed to get a gerbera building outside Docker so I can properly debug now. This will speed up things, especially when doing more risky changes. There are still many places left where I want to apply this identifier(...) method.